crates/astria-sequencer-relayer/tests/blackbox/helper.rs

```diff
@@ -232,7 +331,28 @@ impl TestSequencerRelayerConfig {
             .try_into()
             .unwrap();
 
-        let sequencer = MockServer::start().await;
+        let cometbft = MockServer::start().await;
+
+        let grpc_listener = TcpListener::bind("127.0.0.1:0").await.unwrap();
+        let grpc_addr = grpc_listener.local_addr().unwrap();
+        let sequencer_server_blocks = Arc::new(Mutex::new(VecDeque::new()));
+        let sequencer = MockSequencerServer {
+            blocks: sequencer_server_blocks.clone(),
+        };
+
+        let grpc_server =
+            tonic::transport::Server::builder().add_service(SequencerServiceServer::new(sequencer));
+        let sequencer = tokio::task::spawn(async move {
+            grpc_server
+                .serve_with_incoming(tokio_stream::wrappers::TcpListenerStream::new(
+                    grpc_listener,
+                ))
+                .await
+                .unwrap();
+        });
+
+        // wait for server to start; TODO don't use sleep?
+        tokio::time::sleep(tokio::time::Duration::from_millis(100)).await;
```

Comment:
Is this actually necessary? The server is not immediately ready to use? What if in addition you constructed everything outside of the task and moved it in?
```rust
let sequencer = {
    let serve = grpc_server.serve_with_incoming(TcpListenerStream::new(grpc_listener));
    tokio::task::spawn(async move { serve.await.unwrap() });
};
```